spec/models/validations/setup_validations_spec.rb

```diff
@@ -704,7 +704,7 @@
       end
 
       it "produces no error when location is completes" do
```

Comment:
there's a typo in "completes", not your change but if you can update it that'd be good